Source/WebKit/Scripts/webkit/tests/MessageNames.h

```diff
@@ -145,6 +145,32 @@ constexpr bool messageIsSync(MessageName name)
     return name >= MessageName::TestWithLegacyReceiver_GetPluginProcessConnection;
 }
 
+constexpr bool messageAllowedWhenWaitingForSyncReply(MessageName name)
```

Comment:
Let's not put these implementations in the header.  It'll hurt compile times, unnecessarily increase binary size, and probably not improve performance significantly.